assets/src/edit-story/migration/migrations/v0006_colorToPattern.js

```diff
@@ -17,7 +17,8 @@
 /**
  * External dependencies
  */
-import { parseToRgb } from 'polished';
+import pkg from 'polished';
```

Comment:
Would this be possible? Can we run the `migrate` script via rollup + babel transformer to get a single CommonJS binary. Then we'd not need to do manual transformations here. Occasionally the external deps in `node_modules` would cause some issues, but overall this should all be functional. And we can remove `.js` from the `migrate.js` as well.

Reply:
I haven't looked closely into available options when I wrote this, but I agree that we should avoid having to add `.js` everywhere and changing the `polished` import here.

I can look into using rollup for this if that's the most feasible path forward.